RabbitMQ.Stream.Client/Reliable/RProducer.cs

```diff
@@ -0,0 +1,298 @@
+﻿// This source code is dual-licensed under the Apache License, version
```

Comment:
I think it's OK to keep everything in the `RabbitMQ.Stream.Client` namespace and call this class `ReliableProducer`